git-open.1.md

```diff
@@ -0,0 +1,138 @@
+# git-open - Open the repository's website in your browser.
+
+
+## SYNOPSIS
+
+`git-open` [remote-name] [branch-name]
+
+`git-open` --issue
+
+
+## DESCRIPTION
+
+`git-open` opens the repository's website in your browser. The major well known
+git hosting services are supported.
+
+
+## OPTIONS
+
+
+`-i`, `--issue`
+  Open the current issue. When the name of the current branch matches the right pattern, 
+  it will open the webpage with that issue. See `EXAMPLES` for more information. 
+  This only works on GitHub, GitLab, Visual Studio Team Services and Team Foundation Server at the moment.
+
+`-h`
+  Show a short help text.
+
+
+## EXAMPLES
+
+```sh
+$ git open
+```
+
+It opens https://github.com/TRACKED_REMOTE_USER/CURRENT_REPO/
+
+```sh
+$ git open someremote
+```
+
+It opens https://github.com/PROVIDED_REMOTE_USER/CURRENT_REPO/
+
+```sh
+$ git open someremote somebranch
+```
+
+It opens https://github.com/PROVIDED_REMOTE_USER/CURRENT_REPO/tree/PROVIDED_BRANCH
+
+```sh
+$ git open --issue
+```
+
+If branches use naming convention of `issues/#123`, it opens
+https://github.com/TRACKED_REMOTE_USER/CURRENT_REPO/issues/123
+
+
+## SUPPORTED REMOTE REPOSITORIES
+
+git-open can automatically guess the corresponding repository page for remotes
+(default looks for `origin`) on the following hosts:
+
+- github.com
```

Comment:
Would rather not mention "remotes" but services.. It supports also Github EE, what do others things?

Reply:
I changed it to "Git Hosting Services" in the entire document